{"name": "test_login[standard_user-secret_sauce]", "status": "passed", "attachments": [{"name": "Login", "source": "3ebeefc1-b1b8-49f3-9c87-7078a066118b-attachment.png", "type": "image/png"}], "parameters": [{"name": "username", "value": "'standard_user'"}, {"name": "password", "value": "'secret_sauce'"}], "start": 1707773481954, "stop": 1707773484794, "uuid": "aa3bd1a7-1a4c-42b1-871e-b36d57a30744", "historyId": "77ebba5a14cdba06506b5a8a9cae28a3", "testCaseId": "d3da2953cdfd9c49173b93a1b4e4916c", "fullName": "reports.test_04_pytest_allure_screenshot_error#test_login", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('log_on_failure')"}, {"name": "tag", "value": "login"}, {"name": "parentSuite", "value": "reports"}, {"name": "suite", "value": "test_04_pytest_allure_screenshot_error"}, {"name": "host", "value": "DESKTOP-551JSCE"}, {"name": "thread", "value": "72136-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "reports.test_04_pytest_allure_screenshot_error"}]}